tests/integration/conftest.py

```diff
@@ -78,6 +77,29 @@ def get_log_id(prompt_log_name):
         return match.group(1)
 
 
+def _format_messages(messages):
```

Comment:
What's the thought process here of mocking this out compared to the previous usage that used the actual format_messages call?

Reply:
🤔  The reason why something like this is used in integration tests is that what they do is compare the prompt as created and sent to a mock LLM by the agent, with existing log files of the same thing - files that log prompts. So this is imitating the logging into a file...